Project.toml

```diff
@@ -5,27 +5,27 @@ version = "0.1.0-dev"
 
 [deps]
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearOperators = "5c8ed15e-5a4c-59e4-a42b-c7e8811fb125"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
please add a compat entry for Logging. All packages we use need a compat entry, otherwise the package cannot be registered.

Reply:
done!